src/main/java/Peter/Peter.java

```diff
@@ -0,0 +1,41 @@
+package peter;
+
+import java.util.Scanner;
+
+import utilities.CommandParser;
+import utilities.Storage;
+import utilities.TaskList;
+import utilities.UI;
+
+public class Peter {
+    public static final String FILE_PATH = "tasks/data.txt";
+    private Storage store;
+    private TaskList tl;
```

Comment:
Avoid abbreviation for variables and spell them out in full, "tasks" or "tasklist" would be a better name over here